src/sections/Home/Banner/index.js

```diff
@@ -17,7 +17,7 @@ const RotationalBanner = () => {
   const [Banner, setBanner] = useState(initialValue);
   useEffect(() => {
     if (sessionStorage.getItem("banner")) {
-      sessionStorage.setItem("banner", (1 + Number(Banner)) % BannersList.length);
+      sessionStorage.setItem("banner", (1 ^ Number(Banner)));
```

Comment:
This only works for two banners

Reply:
Oops. That was from my previous commits. Fixed! 👍🏻